### PR TITLE
Fix open contour issue

### LINKED ISSUE #72: contours appear not closed
https://neptun.phys.au.dk/~bassler/DKFZ/DICOM/ example freshly converted to TRIP using dicom2trip 1.3.1. Then loaded with pytripgui:

screenshot_2017-02-25_21-27-10

### PR DESCRIPTION
**Pull request by nbassler**
Closes #72

### CODE REVIEW

**Comment by nbassler** [comment]
pending for new pytrip98 release

**Comment by grzanka** [comment]
pytrip release 1.4.2 triggered https://travis-ci.org/pytrip/pytrip/builds/214756610

File: pytripgui/main.py [hunk 1]
```diff
@@ -230,7 +230,7 @@ def view_about(self, evt):
         info.SetName('PyTRiPGUI')
         info.SetVersion(pytripgui_version)
         info.SetDescription(description)
-        info.SetCopyright('(C) 2012 - 2016 Aarhus Particle Therapy Group')
+        info.SetCopyright('(C) 2012 - 2017 PyTRiP98 Developers')
```
Comment (grzanka) [inline pytripgui/main.py]: My attempt was here:
https://github.com/pytrip/pytripgui/pull/88/files#diff-5a3aa1c8d3bfe0d81a3fcc9f4da31b17

there will be a conflict

Comment (nbassler) [inline pytripgui/main.py]: I rebased.

**Pull request merged**